cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -4,6 +4,10 @@ PACKAGE = "cob_twist_controller"
 from dynamic_reconfigure.parameter_generator_catkin import *
 
 gen = ParameterGenerator()
+interface_enum = gen.enum([ 
```

Comment:
Sounds like an interface in a programming manner. Better would be hw_interface_enum or s.th. like that.